xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserMembershipSheet.xml

```diff
@@ -93,9 +93,13 @@
   #end
   #set($rows = [])
   #foreach ($group in $subGroups)
+    #set ($title = $xwiki.getDocument($group).title)
+    #if ($title == '')
```

Comment:
```suggestion
    #if ("$!title" == '')
```
I would use the silent notation to be extra safe (in case ``getDocument`` returns ``null`` for instance).